doc/concepts/configuration/configuration_etcd.rst

```diff
@@ -6,92 +6,150 @@ Storing configuration in etcd
+The ``tt cluster publish`` command can be used to publish a cluster's configuration to etcd.
 
-    .. code-block:: console
+The example below shows how a :ref:`layout <admin-start_stop_instance-multi-instance-layout>` of the application called ``app`` might look:
 
-        $ etcdctl --user root --password foobar role grant-permission tt readwrite /tt/config/all
-        $ etcdctl --user root --password foobar role grant-permission tt --prefix=true readwrite /tt/
+.. code-block:: none
 
-        $ etcdctl --user root --password foobar user grant-role testuser tt
+    instances.enabled
+    └── app
+        ├── config.yaml
+        ├── cluster.yaml
+        └── instances.yml
 
+*   ``config.yaml`` contains a :ref:`local configuration <etcd_local_configuration>` used to connect to etcd.
+*   ``cluster.yaml`` contains a cluster's configuration to be published.
+*   ``instances.yml`` specifies :ref:`instances <admin-start_stop_instance-multi-instance>` to run in the current environment. ``tt cluster publish`` ignores the configured instances.
 
-    .. code-block:: yaml
+To publish a cluster's configuration (``cluster.yaml``) to an etcd server, execute ``tt cluster publish`` as follows:
 
-        config:
-          etcd:
-            http:
-              request:
-                timeout: 3
-            prefix: /tt
-            endpoints:
-            - http://localhost:2379
-            username: testuser
-            password: foobar
+.. code-block:: console
 
+    $ tt cluster publish "http://localhost:2379/example" instances.enabled/app/cluster.yaml
+
+.. NOTE::
+
+    You can see a cluster's configuration using the :ref:`tt cluster show <tt-cluster>` command.
+
+
+.. _etcd_publishing_configuration_etcdctl:
+
+Publishing configuration using etcdctl
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To publish a cluster's configuration using the ``etcdctl`` utility, use the ``put`` command:
+
+.. code-block:: console
+
+    $ etcdctl put /example/config/all < cluster.yaml
+
+
+
+
+.. _etcd_starting_instances:
+
+Starting Tarantool instances
+----------------------------
+
+To learn how to start Tarantool instances, see the :ref:`Starting Tarantool instances <configuration_run_instance>` section.
+
+
+.. _etcd_reloading_configuration:
+
+Reloading configuration
+-----------------------
+
+By default, Tarantool watches etcd keys with the :ref:`specified prefix <etcd_local_configuration>` for changes in a cluster's configuration and reloads a changed configuration automatically.
+If necessary, you can set the :ref:`config.reload <configuration_reference_config_reload>` option to ``manual`` to turn off configuration reloading:
+
+.. code-block:: yaml
+
+    config:
+      reload: 'manual'
+      etcd:
+        # ...
+
+In this case, you can reload a configuration in the :ref:`application code <configuration_application>` using the ``reload()`` function provided by the :ref:`config <config-module>` module:
```

Comment:
> reload a configuration in the [application code]

I would rather say that one can reload the configuration using an admin console. I guess that it is much more frequent scenario.

Reply:
thanks, extended the description